test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperDescriptorFactoryTest.cs

```diff
@@ -1883,6 +1856,35 @@ public void ValidateTagHelperAttributeDescriptor_WithValidPrefix_ReturnsTrue(str
             return data;
         }
 
+        private static TagHelperDescriptor CreateTagHelperDescriptor(
```

Comment:
There's only 1 `TagHelperDescriptor` constructor used at runtime, the public one. The other internal ones were there for testing convenience.